### ISSUE TITLE
no-actions-hash false positive & failing

### ISSUE THREAD

**Issue by lifeart** [ember-cli/eslint-plugin-ember#2215]
Rule: https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-actions-hash.md
Failing on code like:

```ts
class MyComponent {
     declare actions: typeof dispatchToActions;
}
class MyComponent {
      actions!: typeof dispatchToActions;
}
```

Workaround:

```ts
class MyComponent {
      ['actions']!: typeof dispatchToActions;
}
```

Error: `TypeError: Cannot read properties of null (reading 'type')`

```js
function isActionsProp(property) {
  return (
    types.isIdentifier(property.key) &&
    property.key.name === 'actions' &&
    types.isObjectExpression(property.value) // here property value is null
  );
}
```
